api_tests/nodes/views/test_node_alternative_citation_list.py

```diff
@@ -1,907 +1,760 @@
+        assert len(res.json['errors']) == 1
+        assert res.json['errors'][0]['detail'] == exceptions.NotAuthenticated.default_detail
+        project.reload()
+        assert project.alternative_citations.count() == 0
+
+    #   test_add_no_text_logged_out_private
+        res, project = req(payload_no_text, public=False, logged_out=True, errors=True)
+        assert res.status_code == 401
+        assert len(res.json['errors']) == 1
+        assert res.json['errors'][0]['detail'] == exceptions.NotAuthenticated.default_detail
+        project.reload()
+        assert project.alternative_citations.count() == 0
+
+    def test_add_empty_errors(self, req, payload_empty):
+
+    #   test_add_empty_admin_public
+        res, project = req(payload_empty, is_admin=True, errors=True)
+        assert res.status_code == 400
+        assert len(res.json['errors']) == 2
+        assert res.json['errors'][0]['detail'] == res.json['errors'][1]['detail'], 'This field is required.'
+        project.reload()
+        assert project.alternative_citations.count() == 0
+
+    #   test_add_empty_admin_private
+        res, project = req(payload_empty, public=False, is_admin=True, errors=True)
+        assert res.status_code == 400
+        assert len(res.json['errors']) == 2
+        assert res.json['errors'][0]['detail'] == res.json['errors'][1]['detail'], 'This field is required.'
+        project.reload()
+        assert project.alternative_citations.count() == 0
+
+    #   test_add_empty_non_admin_public
+        res, project = req(payload_empty, errors=True)
+        assert res.status_code == 400
+        assert len(res.json['errors']) == 2
+        assert res.json['errors'][0]['detail'] == res.json['errors'][1]['detail'], 'This field is required.'
+        project.reload()
+        assert project.alternative_citations.count() == 0
+
+    #   test_add_empty_non_admin_private
+        res, project = req(payload_empty, public=False, errors=True)
+        assert res.status_code == 400
+        assert len(res.json['errors']) == 2
+        assert res.json['errors'][0]['detail'] == res.json['errors'][1]['detail'], 'This field is required.'
+        project.reload()
+        assert project.alternative_citations.count() == 0
+
+    #   test_add_empty_non_contrib_public
+        res, project = req(payload_empty, is_contrib=False, errors=True)
+        assert res.status_code == 400
+        assert len(res.json['errors']) == 2
+        assert res.json['errors'][0]['detail'] == res.json['errors'][1]['detail'], 'This field is required.'
+        project.reload()
+        assert project.alternative_citations.count() == 0
+
+    #   test_add_empty_non_contrib_private
+        res, project = req(payload_empty, public=False, is_contrib=False, errors=True)
+        assert res.status_code == 400
+        assert len(res.json['errors']) == 2
+        assert res.json['errors'][0]['detail'] == res.json['errors'][1]['detail'], 'This field is required.'
```

Comment:
Minor; Could make a  var for 'This field is required.' since it shows up a bunch.